test/acceptance/tests.py

```diff
@@ -179,16 +178,14 @@ def submit_self_assessment(self, options=OPTIONS_SELECTED):
         self.self_asmnt_page.assess(options).wait_for_complete()
         self.assertTrue(self.self_asmnt_page.is_complete)
 
-    def _verify_staff_grade_section(self, expected_status, expected_message_title):
+    def _verify_staff_grade_section(self, expected_status):
         """
         Verifies the expected status and message text in the Staff Grade section
         (as shown to the learner).
         """
         self.staff_asmnt_page.wait_for_page()
         self.assertEqual("Staff Grade", self.staff_asmnt_page.label)
         self.staff_asmnt_page.verify_status_value(expected_status)
-        message_title = self.staff_asmnt_page.open_step().message_title
-        self.assertEqual(expected_message_title, message_title)
```

Comment:
`expected_message_title` was the now-irrelevant `STAFF_OVERRIDE_LEARNER_STEPS_NOT_COMPLETE` or `None` for all callers.